DEVELOPMENT.md

```diff
@@ -381,12 +381,69 @@ kubectl set env deployments --all SYSTEM_NAMESPACE=${TARGET_NAMESPACE} -n ${TARG
 ```
 
 This script will cause `ko` to:
+
 - Change (resolve) all `namespace` values in K8s configuration files within the `config/` subdirectory to be updated to a name of your choosing.
 - Builds and push images with the new namespace to your container registry and
 - Update all Tekton Objects in K8s using these images
 
 It will also update the default system namespace used for K8s `deployments` to the new value for all subsequent `kubectl` commands.
 
+---
+
+### Standing up a K8s cluster with Tekton using the `kind` tool
+
+An alternative to standing up your own K8s cluster and installing Tekton using `ko` is by using the [kind](https://kind.sigs.k8s.io/) tool. It was designed to help create and run local,  Kubernetes clusters in Docker to assist in local development and testing.
```

Comment:
nit: looks like there might be a word missing here?  `create and run local,  Kubernetes clusters`

Reply:
I guess that I did not actually need a comma there between local and K8s. Can remove...

Reply:
@sbwsg Fixed